docs/feature_unicode.md

```diff
@@ -97,41 +88,46 @@ You can also switch the input mode by calling `set_unicode_input_mode(x)` in you
 * __UC_WIN__: (not recommended) Windows built-in Unicode input. To enable: create registry key under `HKEY_CURRENT_USER\Control Panel\Input Method\EnableHexNumpad` of type `REG_SZ` called `EnableHexNumpad`, set its value to 1, and reboot. This method is not recommended because of reliability and compatibility issue, use WinCompose method below instead.
 * __UC_WINC__: Windows Unicode input using WinCompose. Requires [WinCompose](https://github.com/samhocevar/wincompose). Works reliably under many (all?) variations of Windows.
 
-### Keycodes
+!> There is an input mode option for BSD, but it's not currently implemented. If you use BSD and would like support for this, please [open an issue on GitHub](https://github.com/qmk/qmk_firmware/issues).
```

Comment:
I moved this section up, before either the keycodes or function options. 

And I reworded it, so that it's more of "asking for help" rather than "asking for a feature request".

```suggestion
```

Reply:
This _is_ up :smile:

Reply:
Ah, ... I see.  Yay for weird git diff handling 😄

Reply:
I see what you mean regarding the “asking for help” bit. I'll change this so that it's more like your version.